src/components/ToggleButtonGroup/ToggleButton.module.css

```
@@ -0,0 +1,35 @@
+.toggle-button{
+  --toggle-button-label-font-size: 16px;
+  --toggle-button-padding-left-right: 10px;
+  --toggle-button-padding-top-bottom: 7px;
+  --toggle-button-border-radius: 0px;
```

Nitpick, but for `0` values, you could omit the `px`.

```suggestion
  --toggle-button-border-radius: 0;
```